projects/aca-playwright-shared/src/page-objects/components/dialogs/create-from-template-dialog-component.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Locator, Page } from '@playwright/test';
+import { BaseComponent } from '../base.component';
+import { timeouts } from '../../../utils';
+
+export class CreateFromTemplateDialogComponent extends BaseComponent {
+  private static rootElement = 'ng-component';
+
+  constructor(page: Page) {
+    super(page, CreateFromTemplateDialogComponent.rootElement);
+  }
+
+  public actionButton = (text: string) => this.getChild('.mat-button .mat-button-wrapper', { hasText: text });
```

Comment:
you don't need the second "wrapper" class here, as that's the internal implementation

Reply:
added data-automation-id for cancel as well as for create button